src/5e-SRD-Features.json

```diff
@@ -19,7 +19,110 @@
     ],
     "url": "/api/features/rage",
-    "feature_specific": {}
+    "feature_specific": {
+      "actions": [
+        {
+          "name": "Rage",
+          "action_type": "bonus",
+          "duration": "10 turns",
+          "prerequisites": [
+            {
+              "type": "boolean",
+              "from": "equipped.armor.heavy"
+            }
+          ],
+          "effects": [
+            {
+              "type": "advantage",
+              "on": {
+                "trigger": "check",
+                "target": "any",
+                "where": {
+                  "check_modifier": {
+                    "index": "str",
+                    "name": "STR",
+                    "url": "/api/ability-scores/str"
+                  }
+                }
+              }
+            },
+            {
+              "type": "advantage",
+              "on": {
+                "trigger": "save",
+                "source": "any",
+                "where": {
+                  "save_modifier": {
+                    "index": "str",
+                    "name": "STR",
+                    "url": "/api/ability-scores/str"
+                  }
+                }
+              }
+            },
+            {
+              "type": "bonus",
+              "amount": 2,
+              "to": "damage",
+              "on": {
+                "trigger": "attack",
+                "source": "self",
+                "target": "any",
+                "where": {
+                  "attack_modifier": {
+                    "index": "str",
+                    "name": "STR",
+                    "url": "/api/ability-scores/str"
+                  }
+                }
+              }
+            },
+            {
+              "type": "resistance",
+              "to": "damage",
+              "target": "self",
+              "source": "any",
+              "where": {
+                "damage_type": {
+                  "index": "bludgeoning",
+                  "name": "Bludgeoning",
+                  "url": "/api/damage-types/bludgeoning"
+                }
+              }
+            },
+            {
+              "type": "resistance",
+              "to": "damage",
+              "target": "self",
+              "source": "any",
+              "where": {
+                "damage_type": {
+                  "index": "piercing",
+                  "name": "Piercing",
+                  "url": "/api/damage-types/piercing"
+                }
+              }
+            },
+            {
+              "type": "resistance",
+              "to": "damage",
+              "target": "self",
+              "source": "any",
+              "where": {
+                "damage_type": {
+                  "index": "slashing",
+                  "name": "Slashing",
+                  "url": "/api/damage-types/slashing"
+                }
+              }
+            },
+            {
+              "spellcasting": false
```

Comment:
What about
```json
{
  "type": "action_restriction",
  "action_types": ["spell"],
  "duration": "10 turns"
}
```

Reply:
ok, with `spellcasting` instead of `spells`. sounds rightier to me